### PR TITLE
Repo updates to v1.8 (unreleased)

### PR DESCRIPTION
**Pull request by cam72cam**
## Target Release

1.8.0

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1_7_0_beta1_feedback.yml
Comment (Yantrio) [inline .github/ISSUE_TEMPLATE/1_7_0_beta1_feedback.yml:1]: I think we want to keep this file for now, so people who go to the issues section have the option there.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,66 +1,18 @@
-## 1.7.0 (Unreleased)
+## 1.8.0 (Unreleased)
 
 UPGRADE NOTES:
-* Backend/S3: The default of `use_legacy_workflow` changed to `false` and is now deprecated. The S3 backend will follow the same behavior as AWS CLI and SDKs for credential search, preferring backend configuration over environment variables. To support the legacy credential search workflow, you can set this option as `true`. It'll be completely removed in a future minor version.
 
 STATE ENCRYPTION
```
Comment (Yantrio) [inline CHANGELOG.md:5]: this line isnt needed anymore

File: .github/ISSUE_TEMPLATE/1_7_0_beta1_feedback.yml
Comment (cam72cam) [reply .github/ISSUE_TEMPLATE/1_7_0_beta1_feedback.yml:1]: We are already on the RC, is it still useful?

**Comment by cam72cam** [comment]
Will merge after https://github.com/opentofu/opentofu.org/pull/291

**Pull request merged**